app/controllers/spots_controller.rb

```diff
@@ -23,6 +23,12 @@ def create
     end
   end
 
+  def destroy
+    @spot = Spot.find(params[:id])
+    @spot.destroy
+    redirect_to spot_paths
```

Comment:
should be spots_path right? when you destroy the spot the spot_path doesn't exist anymore. (be careful path is singular)